### ISSUE TITLE
Depletion ignored in double-negative event

### ISSUE THREAD

**Issue by bgyori** [clulab/reach#538]
The stem of "deplete", "depletion", etc. should be added to this line:
https://github.com/clulab/reach/blob/38d2c4330e2ec8064a2829288a3c7b42b844eeef/main/src/main/scala/org/clulab/reach/darpa/DarpaActions.scala#L361
to correctly parse a sentence like "EEEB reduced GSH depletion".

**Comment by MihaiSurdeanu** [comment]
I agree!

@hickst: can you please add "deplet" to the regex?
@bgyori: any other obvious ones we're missing?

**Comment by myedibleenso** [comment]
Makes sense, but let's review a couple of examples of shell output to ensure this change looks good to everyone.

**Comment by hickst** [comment]
I'm not sure what are expected results for this issue. Ignoring the fact that Reach doesn't recognize EEEB (what is it?)....If I type any of the following into the Reach shell they all produce the same result: a `Positive_activation` mention via the `Negative_activation_syntax_1_verb` rule which recognizes only `GSH` as the 'controlled' entity:
```
ATP reduced GSH depletion.
ATP reduced GSH deletion.
ATP reduced GSH deactivation.
```

**Comment by myedibleenso** [comment]
@hickst, that sounds right.  I would expect the polarity to be flipped (reducing a reduction).

**Comment by hickst** [comment]
OK, thanks!

**Comment by hickst** [comment]
Issue closed by pull request #539
